leptos_macro/src/view.rs

```diff
@@ -1102,30 +1095,29 @@ fn element_to_tokens(
                     false,
                 ),
                 Node::Text(node) => {
-                    if let Some(primitive) = value_to_string(&node.value) {
-                        (quote! { #primitive }, true)
-                    } else {
-                        let value = node.value.as_ref();
-                        (
-                            quote! {
-                                #[allow(unused_braces)] #value
-                            },
-                            false,
-                        )
-                    }
+                    (quote! { #node }, true)
+                }
+                // TODO: Implement html  escaping?
```

Comment:
I don't believe HTML escaping needs to be added here or on `Node::Text`. These static string types are escaped by the SSR optimization code for SSR, and don't need to be escaped in the browser because they're created using `document.createTextNode()` not an HTML element. I may be wrong.